fuzz.go

```diff
@@ -0,0 +1,25 @@
+// +build gofuzz
+
+package sftp
+
+import "bytes"
+
+type sink struct{}
+
+func (*sink) Close() error                { return nil }
+func (*sink) Write(p []byte) (int, error) { return len(p), nil }
+
+var devnull = &sink{}
+
+// To run: go-fuzz-build && go-fuzz
+func Fuzz(data []byte) int {
+	c, err := NewClientPipe(bytes.NewReader(data), devnull)
+	if err == nil {
+		c.Close()
+		return 1
+	}
+	if c != nil {
+		panic("error, but non-nil *Client")
```

Comment:
Please drop this check, there is no utility in fuzzing if new client pipe returns a client instance in the case of an error.

Reply:
Done.